src/App.js

```diff
@@ -1,7 +1,19 @@
-import './App.css';
-import {RoutesPath} from './Routes'
-
+import { useEffect } from "react";
+import { useReducerContext } from './context/context';
+import { RoutesPath } from './Routes'
+import { getProductsData } from './serverCalls/getProductsData';
 function App() {
+  const { dispatch } = useReducerContext();
```

Comment:
Can name something more specific instead of `useReducerContext` if this context is specific to just products.

Reply:
Made the changes in next PR